forge/ee/routes/billing/index.js

```diff
@@ -100,8 +106,8 @@ module.exports = async function (app) {
                 break
             case 'checkout.session.expired':
                 // should remove the team here
-                console.log('checkout.session.expired')
-                console.log(event)
+                app.log.info('checkout.session.expired')
```

Comment:
Can we add some more context in the message so we can relate it back to a user/team/stripe something?